x/concentrated-liquidity/client/cli/tx.go

```diff
@@ -71,3 +72,13 @@ func NewCollectIncentivesCmd() (*osmocli.TxCliDesc, *types.MsgCollectIncentives)
 		Flags:               osmocli.FlagDesc{RequiredFlags: []*flag.FlagSet{FlagSetJustPoolId()}},
 	}, &types.MsgCollectIncentives{}
 }
+
+func NewCreateIncentiveCmd() (*osmocli.TxCliDesc, *types.MsgCreateIncentive) {
+	return &osmocli.TxCliDesc{
+		Use:                 "create-incentive [incentive-denom] [incentive-amount] [emission-rate] [start-time] [min-uptime]",
+		Short:               "create an incentive record to emit incentives (per second) to a given pool",
+		Example:             "create-incentive uosmo 69082 0.02 100 2023-03-03 03:20:35.419543805 24h --pool-id 1 --from val --chain-id osmosis-1",
+		CustomFlagOverrides: poolIdFlagOverride,
+		Flags:               osmocli.FlagDesc{RequiredFlags: []*flag.FlagSet{FlagSetJustPoolId()}},
+	}, &types.MsgCreateIncentive{}
```

Comment:
Not blocking, but we have been trying to move away from mandatory flags, would be nice if we could just make poolId be part of the create incentive message